lib/src/base-styles/_nx-grid.scss

```diff
@@ -17,7 +17,7 @@
 .nx-grid-col {
   @include nx-container-helpers.container-vertical;
 
-  border-left: var(--nx-border-default);
+  border-left: var(--nx-border-2x);
```

Comment:
The vertical lines actually _do_ have a border radius to them in sketch, which is problematic as we can't do that on a border.

Reply:
> The vertical lines actually _do_ have a border radius to them in sketch, which is problematic as we can't do that on a border.

You're right. Not sure how we would want to go about this. 
Thoughts about this @alexalyons126, @glenhunter, @entroform  ?

Reply:
> > The vertical lines actually _do_ have a border radius to them in sketch, which is problematic as we can't do that on a border.
> 
> You're right. Not sure how we would want to go about this.
> Thoughts about this @alexalyons126, @glenhunter, @entroform ?

As per discussion, the `border-radius` is barely visible on a `2px` line, so we are fine with current implementation of having no `border-radius`.